packages/core/changelog/@unreleased/pr-6865.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+fix:
+  description: '[core] a11y(Tooltip): wrap contents in "tooltip" aria role'
```

Comment:
✂️

Reply:
> lemme know if you think the `hostNodes` change is important### 

This PR is a precursor to https://github.com/palantir/blueprint/pull/6896 , where this change is more beneficial